tests/integration/hugectr/test_nvt_hugectr_training.py

```diff
@@ -379,3 +381,20 @@ def _convert(data, slot_size_array):
     dense = []
 
     return dense, cat, row_ptrs
+
+
+DIR = "/model/"
+DATA_DIR = DIR + "data/"
+
+
+def test_server():
+    tri_cmd = [
+        "tritonserver",
+        "--model-repository=/model/models/",
+        "--backend-config=hugectr,test_model=/model/models/test_model/1/model.json",
+        "--backend-config=hugectr,supportlonglong=true",
+        "--model-control-mode=explicit",
+        "--load-model=test_model_ens",
+    ]
+    subprocess.Popen(tri_cmd)
```

Comment:
What does this test?  I don't think this test will fail if the tritonserver fails to start. 

Also does this test leave a dangling tritonserver process running ? (like how is the process cleaned up)? 

also https://github.com/NVIDIA/NVTabular/pull/863/files#discussion_r646727169